cilium-dbg/cmd/shell.go

```diff
@@ -84,10 +85,7 @@ func shellExchange(w io.Writer, format string, args ...any) error {
 			return nil
 		case "[stdout]":
 		default:
-			if _, err := w.Write(line); err != nil {
-				return err
-			}
-			if _, err := w.Write(newline); err != nil {
+			if _, err := fmt.Fprintf(w, "%s\n", line); err != nil {
```

Comment:
Could be simplified to:

```suggestion
			if _, err := fmt.Fprintln(w, line); err != nil {
```